packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -68,8 +68,8 @@ export const useButtonState = (state: ButtonState): ButtonState => {
   };
 
   // Set the aria-disabled and disabled props correctly.
-  state['aria-disabled'] = disabled /*|| disabledFocusable*/;
-  state.disabled = as === 'button' ? disabled /* && !disabledFocusable*/ : undefined;
+  state['aria-disabled'] = disabled || disabledFocusable;
```

Comment:
I don't think we should set `aria-disabled` on disabled button elements. If you switch the order, you could do:

```
state.disabled = as === 'button' ? disabled && !disabledFocusable : undefined;
state['aria-disabled'] = disabled && !state.disabled;
```

Reply:
Done, I went and made the change.